### PR TITLE
Improved known bug descriptions

### PR DESCRIPTION
**Pull request by omichel**
👍

### CODE REVIEW

File: guide/general-bugs.md [hunk 1]
```diff
@@ -37,3 +44,16 @@ on another object in some direction than in some other, even if the friction
...
+### Remote desktop
+
+Webots is not guaranteed to work through a remote desktop application.
+This is because Webots has strong ties with the local graphics card for on-screen and off-screen OpenGL rendering.
+Unfortunately, several remote desktop applications do not support this very well.
+
+### Virtualization
+
+Because it highly relies on OpenGL, Webots may not work properly in virtualized
+environments (such as VMWare or VirtualBox) which often lack a good OpenGL
```
Comment (tn12787) [inline guide/general-bugs.md]: you should remove the "a" before "OpenGL support",

File: guide/mac-os.md [hunk 2]
```diff
@@ -1,9 +1,6 @@
...
 
-### MATLAB and robot plugins
+### MSAA anti-aliasing may corrupt movie creation
 
-The controllers and the robot plugins (e.g. the robot windows) are sharing the
-same process. Generally this mechanism works well, but some instabilities
-(crashes and warnings) may occur when a robot uses a MATLAB controller and a
-native robot window at the same time. This is due to some conflicts between the Qt
-libraries used by MATLAB and the one used by Webots.
+On some macOS configuration, it was observed that activating the MSAA anti-aliasing was causing problems with movie creation.
```
Comment (tn12787) [inline guide/mac-os.md]: configuration -> configurations, also the "the" before "MSAA anti-aliasing" should be removed too.

File: guide/mac-os.md [hunk 3]
```diff
@@ -1,9 +1,6 @@
...
+If you experience any problem with movie creation, try to deactivate the Main 3D view anti-aliasing from the Webots OpenGL preferences.
```
Comment (tn12787) [inline guide/mac-os.md]: problem -> problems

Comment (tn12787) [reply guide/mac-os.md]: also the "the" before "Main 3D view antialiasing" should go too.

File: guide/mac-os.md (hunk 2 above)
Comment (stefaniapedrazzi) [inline guide/mac-os.md]: configuration ->configurations

File: guide/general-bugs.md [hunk 4]
```diff
@@ -1,32 +1,39 @@
...
 website](http://intellinuxgraphics.org).
 
-### Virtualization
+### NAOqi support is discontinued: NAQqi-based controllers may not work properly
 
-Because it highly relies on OpenGL, Webots may not work properly in virtualized
-environments (such as VMWare or VirtualBox) which often lack a good OpenGL
-support. Hence, Webots may exhibit some display bugs, run very slowly or crash
-in such environments.
+The makers of the NAO humanoid robot, SoftBank Robotics, do not maintain any more the NAOqi interface library that is used to connect Webots to NAOqi and Choregraphe.
```
Comment (tn12787) [inline guide/general-bugs.md]: do not maintain any more -> no longer maintain

**Comment by omichel** [comment]
👍

**Pull request merged**